### PR TITLE
navy url now reads the current url from the container state

### PR DESCRIPTION
**Pull request by tanuck**
Because previously navy used to interpolate the url for a service from the name and external IP, there were some scenario's where the incorrect url could be returned (see #58). Url get and creation are now seperate with the latter inspecting the running container.

### CODE REVIEW

**Comment by madjam002** [comment]
This looks awesome. Do you think though that `getUrlFromService` in `util` should return `null` rather than `'-'` when there is no virtual host?

**Comment by tanuck** [comment]
Yeah thats perfectly reasonable. I just kept the bahavior the same as `navy ps` for now, but I guess we could catch that scenario in `src/cli/ps.js`?

**Comment by madjam002** [comment]
Yeah exactly, something simple like `getUrlFromService(service) || '-'` or even better:

```js
const getDisplayUrl = serviceUrl => serviceUrl != null ? serviceUrl : '-'
```

File: packages/navy/src/util/nipio.js [hunk 1]
```diff
@@ -2,6 +2,8 @@
```
Comment (ThomWright) [inline packages/navy/src/util/nipio.js]: Could we rename this file to something like `service-host.js`?

Comment (tanuck) [inline packages/navy/src/util/nipio.js]: Yeah sure

**Comment by tanuck** [comment]
Happy to merge?

**Pull request merged**